spec/controllers/api/v1/items_find_controller_show_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Items::FindController, type: :controller do
```

Comment:
You have to specify the `type: :controller` to tell it what kind of test this is.